examples/intel/user/password_breached/src/main/java/cloud/pangeacyber/examples/App.java

```diff
@@ -0,0 +1,38 @@
+package cloud.pangeacyber.examples;
+
+import cloud.pangeacyber.pangea.intel.UserIntelClient;
+import cloud.pangeacyber.pangea.intel.models.*;
+import cloud.pangeacyber.pangea.exceptions.ConfigException;
+import cloud.pangeacyber.pangea.Config;
+
+public class App
+{
+    public static void main( String[] args )
+    {
+        Config cfg = null;
+        try {
+            cfg = Config.fromEnvironment(UserIntelClient.serviceName);
+        } catch (ConfigException e){
+            System.out.println(e);
+            System.exit(1);
+        }
+
+        UserIntelClient client = new UserIntelClient(cfg);
+        UserPasswordBreachedResponse response = null;
+        try {
+            response = client.breached(
+                new UserPasswordBreachedRequest.UserPasswordBreachedRequestBuilder(HashType.SHA256, "5baa6")
```

Comment:
we should show the user how to generate the hash prefix then how to use the results.